README.md

```diff
@@ -76,15 +77,15 @@ The archive, downloaded from github, contains a simple playbook that just uses t
 
 Use the steps below (dev mode), to quickly exercise the API  
 1. Get the list of available playbooks (should just be test.yml)  
-```curl -k -i https://localhost:5001/api/v1/playbooks  -X GET```
+```curl -k -i -H "Authorization: eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJleHAiOjE1MzczODA3MTR9.CbTXvBum5mCq9s56wJNiMn8JLJ0UzzRdwdeOFctJtbI" https://localhost:5001/api/v1/playbooks  -X GET```
```

Comment:
Shouldn't the readme show the login step and describe the use of the token?

Reply:
+1